proposals/0021/README.md

```diff
@@ -1,30 +1,212 @@
+periodically scrape. This is an example of metadata file:
+
+```yaml
+# Artifact Hub package metadata file
+version: A SemVer 2 version (required)
+name: The name of the package (only alphanum, no spaces, dashes allowed) (required)
+displayName: The name of the package nicely formatted (required)
+createdAt: The date this package was created (RFC3339 layout) (required)
+description: A short description of the package (required)
+logoPath: Path to the logo file relative to the package directory (optional, but it improves package visibility)
+digest: String that uniquely identifies this package version (optional)
+license: SPDX identifier of the package license (https://spdx.org/licenses/) (optional)
+homeURL: The URL of the project home page (optional)
+appVersion: The version of the app that this contains (optional)
+containersImages: # (optional)
+  - name: Image identifier (optional)
+    image: The format should match ${REGISTRYHOST}/${USERNAME}/${NAME}:${TAG}
+    whitelisted: When set to true, this image won't be scanned for security vulnerabilities
+containsSecurityUpdates: Whether this package version contains security updates (optional, boolean)
+operator: Whether this package is an Operator (optional, boolean)
+deprecated: Whether this package is deprecated (optional, boolean)
+keywords: # (optional)
+  - A list of keywords about this package
+  - Using one or more categories names as keywords will improve package visibility
+links: # (optional)
+  - name: Title of the link (required for each link)
+    url: URL of the link (required for each link)
+readme: | # (optional)
+  Package documentation in markdown format
+
+  Content added here will be rendered on Artifact Hub
+install: | # (optional)
+  Brief install instructions in markdown format
+
+  Content added here will be displayed when the INSTALL button on the package details page is clicked.
+changes: # (optional)
+  - A list of changes introduced in this package version
+  - Use one entry for each of them
+maintainers: # (optional)
+  - name: The maintainer name (required for each maintainer)
+    email: The maintainer email (required for each maintainer)
+provider: # (optional)
+  name: The name of the individual, company, or service that provides this package (optional)
+ignore: # (optional, entries use .gitignore syntax)
+  - lib
+```
+
+https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+
+Having a clear separation between the metadata used by ArtifactHub and the
+actual artifact (the image itself) will sound a complication, but it gives us
+more control and flexibility between ongoing work (PRs on specific actions) and
+their release cycle (ActionHub artifact).
+
 ## System-context-diagram
 
+A contributor will interface with a traditional repository, currently located in
+[github.com/gianarb/hub](https://github.com/gianarb/hub), it will be moved to its
+permanent location [github.com/gianarb/hub](https://github.com/tinkerbell/hub).
```

Comment:
this is the only repo that artifact hub will scrape?

Reply:
I think you mentioned out-of-band that artifacthub can also display groupings of different repos via keywords or something right? It would be nice to have that as a one-liner for future expansion.

Reply:
Everybody in the world is free to add more repositories with their actions. ArtifactHub scrapes them all. Every package has an org as "owner", ours is called tinkebell-community.